README.md

```diff
@@ -1 +1,156 @@
-# opencraft
+OpenCraft
+=========
+
+Install
+-------
+
+Instructions based on Ubuntu 14.04.
+
+Install the system package dependencies & virtualenv:
+
+```
+$ sudo apt-get install `cat debian_packages.lst`
+$ pip3 install --user virtualenv && pip3 install --user virtualenvwrapper
+```
+
+Ensure you load virtualenv with Python 3 in `~/.bashrc`:
+
+```
+export PATH="$PATH:$HOME/.local/bin" VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+source $HOME/.local/bin/virtualenvwrapper.sh
+```
+
+Then reload `~/.bashrc`, create the virtual env and install the Python requirements:
+
+```
+$ . ~/.bashrc
+$ mkvirtualenv -p /usr/bin/python3 opencraft
+$ pip install -r requirements.txt
+```
+
+
+Configure
+---------
+
+Create an `.env` file at the root of the repository or set environment variables, customizing the
+settings from `opencraft/settings.py` which are loaded via `env()`.
```

Comment:
Since many settings have no defaults and must be set in this file, it would be nice to have a working example `.env` development configuration in the readme.